README.md

```diff
@@ -1,5 +1,22 @@
 # DevOps Apprenticeship: Project Exercise
 
+### Environment variables
```

Comment:
We should also add some documentation to record the fact that the app is hosted on Azure; e.g. where it's hosted, and how the pipeline deploys it